tests/github_services_test.py

```diff
@@ -0,0 +1,235 @@
+                    "discussionCategories": {
+                        "nodes": [
+                            {
+                                "id": "test_category_id_1",
+                                "name": "test_category_name_1",
+                                "repository": {
+                                    "discussions": {
+                                        "edges": [
+                                            {
+                                                "node": {
+                                                "id": "test_discussion_id_1",
+                                                "title": "test_discussion_title_1"
+                                                }
+                                            }
+                                        ]
+                                    }
+                                }
+                            },
+                            {
+                                "id": "test_category_id_2",
+                                "name": "test_category_name_2",
+                                "repository": {
+                                    "discussions": {
+                                        "edges": [
+                                            {
+                                                "node": {
+                                                "id": "test_discussion_id_2",
+                                                "title": "test_discussion_title_2"
+                                                }
+                                            }
+                                        ]
+                                    }
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+        self.response_for_comment: Dict[str, Any] = {
+            "data": {
+                "addDiscussionComment": {
+                    "clientMutationId": 'test_id',
+                    "comment": {
+                        "id": "test_discussion_id_1"
+                    }
+                }
+            }
+        }
+        self.pull_response: List[Dict[str, Any]] =  [{
+            'html_url': 'https://githuburl.pull/123',
+            'number': 123,
+            'title': 'PR title 1',
+            'user': {
+                'login': 'authorName',
+            },
+            'assignees': [{
+                'login': 'reviewerName1',
+            }, {
+                'login': 'reviewerName2',
+            }]
+        }, {
+            'html_url': 'https://githuburl.pull/234',
+            'number': 234,
+            'title': 'PR title 2',
+            'user': {
+                'login': 'authorName',
+            },
+            'assignees': [{
+                'login': 'reviewerName1',
+            }, {
+                'login': 'reviewerName2',
+            }]
+        }]
+        def get_past_time(hours: int=0) -> str:
```

Comment:
Pull this into a separate top-level private _get_past_time() method within the class, and call it using _get_past_time(...). Try not to define methods within other methods.

Reply:
done